.github/workflows/release.yml

```diff
@@ -11,7 +11,7 @@ on:
     branches:
       - main
     tags:
-      - '*'
+      - v[0-9]+.[0-9]+.[0-9]+
```

Comment:
Does this restrict the versioning to between 0 and 9?
If so, why is it placed? Semantic versioning allows for values above 9 as well. Good example is python 3.11.

Reply:
[0-9]+ is any number of digits, so this should be fine, no? I tested it, however, only for the single-digit case

Reply:
I am not knowing much about regex but I looked it up and it seems good